examples/rac-spectrum-tailwind/src/components/GenInputField.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { Input, Group, TextField, Button } from "react-aria-components";
+
+export function GenInputField() {
+  let [value, setValue] = useState("");
+  let [isTextFieldFocused, setIsTextFieldFocused] = useState(false);
+  return (
+    <Group
```

Comment:
this example seems inverted from the docs https://react-spectrum.adobe.com/react-aria/Group.html#example

I think you could stop tracking the focused state if you flipped it around

Reply:
I wanted to have the focus ring around the entire bar if the input was focused, and just around the button if it is focused. I think that's different behavior than what you're talking about (I may be wrong).